pepper/cli.py

```diff
@@ -431,15 +438,15 @@ def run(self):
                 api.req('/stats')
             except Exception as e:
                 if e.args[0] is not 2:
-                    logger.error('Unable to load login token from ~/.peppercache '+str(e))
+                    logger.error('Unable to load login token from {} {}'.format(token_file, str(e)))
```

Comment:
Will you please index these substitutions so it works on 2.6?

Reply:
Thanks for your quick review. I update the PR now.